src/pages/overlay/components/buttons/buttons.module.scss

```
@@ -7,27 +7,31 @@
   margin: 3rem 0 0;
   z-index: 1; // stay above the popups
 
-  button {
+  .btn,
+  .tooltip {
     display: flex;
     justify-content: center;
     align-items: center;
     width: 4rem;
     height: 4rem;
-    border-radius: 0.5rem;
-    background: $primary-color;
-    cursor: pointer;
-    box-shadow: 0 0.125rem 0.25rem $accent-color;
-    transition: $transition;
-    transition-property: outline, filter;
 
-    img {
-      height: 60%;
-    }
+    button {
```

```suggestion
    button {
      width: 100%;
      height: 100%;
      display: flex;
      justify-content: center;
      align-items: center;
```
The button is what contains the image, and so should have the flex alignment

---

ah, thanks a ton again Matt this makes 100% sense. I was getting some problems with the image not taking the correct size inside the tooltip, but the height and width 100 along with the flex was the solution